### PR TITLE
DOCSP-17133 update FAQ

### PR DESCRIPTION
**Pull request by kyuan-mongodb**
### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-17133

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/java/docsworker-xlarge/DOCSP-17133-UpdateFAQ/faq/#do-i-have-to-specify-an-id-field-value-myself-

### Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Does it render on staging correctly?
- [x] Are all the links working?
- [x] Are the staging links in the PR description updated?

### If your page documents a concept, does it meet the following criteria?

- [ ] Target the [Jasmin persona](https://drive.google.com/file/d/14FbBOLCVxwSP6M9BK4Nz1Ir9tzxT8_02/view)
- [ ] Target the [Lucas persona](https://drive.google.com/file/d/1J2vqJxo7ldv7OP_obA9Q-avf0o_ju4Lk/view)

### CODE REVIEW

File: source/faq.txt [hunk 1]
```diff
@@ -47,16 +47,7 @@ POJOs
...
-
-.. code-block:: java
-
-   public class Entity {
-      public final ObjectId id;
-      public String value;
-
-      public Entity(){ }
-   }
+No, the PojoCodec automatically generates an ``ObjectId``.
```
Comment (biniona-mongodb) [inline source/faq.txt:50]:
```suggestion
No, the PojoCodec automatically generates an ObjectId.
```

Question: Should ObjectId not be monospaced? [On the ObjectID manual page](https://docs.mongodb.com/manual/reference/method/ObjectId/) it seems like ObjectId is not monospaced.

**Review by biniona-mongodb — CHANGES_REQUESTED** [review]
Two items to discuss, then should be good to go through.

File: source/faq.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (kyuan-mongodb) [reply source/faq.txt:50]: I believe you have the correct suggested replacement -- On the [POJO page](https://docs-mongodbcom-staging.corp.mongodb.com/java/docsworker-xlarge/DOCSP-16491-Typos-After-Crud/fundamentals/data-formats/document-data-format-pojo/#configure-the-driver-for-pojos), its also called a `PojoCodecProvider`

**Review by biniona-mongodb — APPROVED** [review]
LGTM

**Pull request merged**